server/src/modules/auth/auth.service.ts

```diff
@@ -16,6 +17,11 @@ export class AuthService {
     tokenResponse.token = jwt.sign(tokenPayload, process.env.JWT_SECRET, {
       expiresIn: '24h',
     });
+    tokenResponse.refreshToken = jwt.sign(
+      { sub: user._id },
```

Comment:
If you use different payload, you should declare interface and use type checking.